sdk/communication/azure-communication-rooms/tests/conftest.py

```diff
@@ -0,0 +1,54 @@
+# --------------------------------------------------------------------------
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# The MIT License (MIT)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the ""Software""), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#
+# --------------------------------------------------------------------------
+import pytest
+import os
+from devtools_testutils import add_general_regex_sanitizer, add_header_regex_sanitizer, add_body_key_sanitizer, set_default_session_settings
+from azure.communication.rooms._shared.utils import parse_connection_str
+
+@pytest.fixture(scope="session", autouse=True)
+def add_sanitizers(test_proxy):
+    set_default_session_settings()
+
+    communication_connection_string = os.getenv("COMMUNICATION_CONNECTION_STRING_ROOMS", "endpoint=https://sanitized.communication.azure.com/;accesskey=fake===")
+
+    add_general_regex_sanitizer(regex=communication_connection_string, value="endpoint=https://sanitized.communication.azure.com/;accesskey=fake===")
```

Comment:
Nit: when sanitizing strings directly, like environment variable values, I recommend using `add_general_string_sanitizer` instead (sometimes strings accidentally contain regex expressions and cause issues when using a regex sanitizer)

Reply:
Done, thanks for the suggestion!